code-challenges.test.js

```diff
@@ -24,6 +24,26 @@ const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "s
 
 // Pseudo code:
 
+// const shuffleArray = (array) => {
+// array.shift()
+//  const shuffle = array.sort(() => Math.random())
+//  return shuffle
+// }
+
+// const newColors = shuffleArray([...colors1])
+
+// describe("shuffleArray", () => {
+//     it("removes the first item from the array and shuffles the remaining items", () => {
+//         expect(["purple", "blue", "green", "yellow", "pink"]).toEqual(expect.shuffleArray(colors1))
+//         expect(["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]).toEqual(expect.shuffleArray(colors2))
+    
+//     })
+// })
```

Comment:
I think you had the right idea, but the documentation threw you off.  Be sure to run your tests!